QuickChat/Configuration/ButtonsFrame.lua

```diff
@@ -0,0 +1,38 @@
+local frame = CreateFrame("Frame", "QuickChat_BtnCfgFrame", QuickChat.mainConfigurationFrame)
+frame.name = "Buttons"
+frame.parent = QuickChat.mainConfigurationFrame.name
+InterfaceOptions_AddCategory(frame)
+QuickChat.buttonsConfigurationFrame = frame
+
+function frame:OnAddonLoaded()
+    local l = QuickChat_GetLocalization()
+    
+    local info = frame:CreateFontString("QuickChat_BtnCfgFrame_Info", "OVERLAY", "GameFontHighlight")
+    info:SetPoint("TOP", frame, "TOP", 0, -40)
+    info:SetText(l["INFO"])
```

Comment:
Assuming you want to add "Help" text for "Buttons" page, this "INFO" need to be into something meaningful.